src/index.d.ts

```diff
@@ -65,6 +65,11 @@ declare interface PathInfo {
    * original extension as passed via request
    */
   originalExtension: string;
+
+  /**
+   * original path as passed before folder mapping
+   */
+  mappedPath: string;
```

Comment:
Hate to make another prop, wasn't sure though. Should I have just overridden `originalPath`?

Reply:
True... I like `unmappedPath`.